site-example.yml

```diff
@@ -4,10 +4,15 @@ Site: "Site Title"                                                      # Requir
 Output: "docs"                                                          # Required - Output directory where html files are generated
 Logo: "https://<domain.com>/url-to-logo.svg"                            # Site Branding
 SEO Author: "Author Name"                                               # Optional - Meta author tag for SEO
+Language: "en"                                                          # Required - Language of the docs
 
-Language:                                                               # Optional - Translation links
-  - Chinese: https://<domain.com>/cn
-  - Japanese: https://<domain.com>/jp
+OtherLanguages:                                                         # Optional - Translation links
```

Comment:
I am not for changing the structure. How about we infer the Language. We already know if its english or chinese or japanese language.

Reply:
@deekoder do you mean we can infer the language from the name of the file? Like site.yaml is english, site_CN.yml is chinese and so on.